XCUITests/FxScreenGraph.swift

```diff
@@ -99,11 +100,18 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
             }
 
             scene.tap(startBrowsingButton, to: NewTabScreen)
+            scene.tap(app.buttons["Sign in to Firefox"], to: FxSyncLogin)
         }
 
         i += 1
     }
 
+    map.createScene(FxSyncLogin) { scene in
+        scene.backAction = {
+            app.buttons["Cancel"].tap()
```

Comment:
> you said three options, but there are only two, what is the third one

You're right!

I think the third would be: don't link to the sign up page in `FxScreenGraph` at all.